docs/BR.md

```diff
@@ -2857,33 +2854,17 @@ This section contains several fields that are common among multiple certificate
 
 ##### 7.1.2.11.2 CRL Distribution Points
 
-If present, the CRL Distribution Points extension MUST be formatted as follows:
-
-Table: `CRLDistributionPoints` profile
-
-| __Field__                   | __Presence__    | __Description__ |
-| ---                         | --              | ------          |
-| `CRLDistributionPoints`     |                 |                 |
-| \ \ **1**                   | MUST            | The first `DistributionPoint` present in the `CRLDistributionPoints` |
-| \ \ \ \ `distributionPoint` | MUST            | The `DistributionPointName` MUST be a `fullName` formatted as described below. |
-| \ \ \ \ `reasons`           | MUST NOT        |                 |
-| \ \ \ \ `cRLIssuer`         | MUST NOT        |                 |
-| \ \ **2+**                  | NOT RECOMMENDED | Additional `DistributionPoint`s are NOT RECOMMENDED. |
-| \ \ \ \ `distributionPoint` | MUST            | The `DistributionPointName` MUST be a `fullName` formatted as described below. |
-| \ \ \ \ `reasons`           | MUST NOT        |                 |
-| \ \ \ \ `cRLIssuer`         | MUST NOT        |                 |
-| \ \ **3**                   | MUST NOT        | `DistributionPoints` that do not conform to the above requirements MUST NOT be present. |
-
-Table: `fullName` profile
-
-| __Field__                           | __Presence__ | __Description__ |
-| ---                                 | -            | -----           |
-| `fullName`                          |              |                 |
-| \ \ **1**                           | MUST         | The first `GeneralName` present in `fullName` MUST be of type `uniformResourceIdentifier` |
-| \ \ \ \ `uniformResourceIdentifier` | MUST         | The HTTP URL of the Issuing CA's CRL service for this certificate. |
-| \ \ **2+**                          | MAY          | Additional `GeneralName`s MAY be present. If present, they MUST be of type `uniformResourceIdentifier`. |
-| \ \ \ \ `uniformResourceIdentifier` | MUST         | If present, the scheme of the `uniformResourceIdentifier` MUST be "http". |
-| \ \ **3**                           | MUST NOT     | `GeneralName`s that do not conform to the above requirements MUST NOT be present. |
+If present, the CRL Distribution Points extension MUST contain at least one `DistributionPoint`; containing more than one is NOT RECOMMENDED. All `DistributionPoint` items must be formatted as follows:
```

Comment:
Justification for change:

```The tables in 7.1.2.11.2 seem confusing. Personally, I would include tables profiling the inner DistributionPoint and uniformResourceIdentifier objects, and use non-tabulated language to describe the fact that including more than one entry in the outer CRLDistributionPoints and fullName objects is NOT RECOMMENDED.```

Comment: I think this reads well and maintains the original intent of the tabular format.